dimod/serialization/coo.py

```diff
@@ -12,9 +12,69 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
-# ================================================================================================
+# =============================================================================
+"""
+A simple text encoding of dimod BQMs.
+
+The COOrdinate_ list is a sparse matrix representation which can be used to
+store binary quadratic models. This format is best used when readability is
+important.
+
+Note that this format only works for BQMs that are labelled with positive
+integers.
+
+.. _COOrdinate: https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO)
+
+Examples:
+
+    >>> from dimod.serialization import coo
+
+    Serialize a QUBO
+
+    >>> Q = {(0, 0): -1, (0, 1): 1, (1, 2): -4.5}
+    >>> bqm = dimod.BinaryQuadraticModel.from_qubo(Q)
+    >>> print(coo.dumps(bqm))
+    0 0 -1.000000
+    0 1 1.000000
+    1 2 -4.500000
+
+    We can also include the vartype as a header
+
+    >>> Q = {(0, 0): -1, (0, 1): 1, (1, 2): -4.5}
+    >>> bqm = dimod.BinaryQuadraticModel.from_qubo(Q)
+    >>> print(coo.dumps(bqm, vartype_header=True))
+    # vartype=BINARY
+    0 0 -1.000000
+    0 1 1.000000
+    1 2 -4.500000
+
+    Loading the same, note that we must specify the vartype
```

Comment:
Ungrammatical. I think you want something like "When loading, you must specify the vartype."